modules/insights-operator-using-filters-on-advisor-recommendations.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="insights-operator-using-filters-on-advisor-recommendations_{context}"]
+= Using filters to remove unnecessary results from Insights advisor recommendations
+
+As an OpenShift cluster manager, you can filter the recommendations that are displayed on the recommendations list. You might do this to reduce the number of reported recommendations or to concentrate on your highest priority issues. 
+
+Use the following example to select from several categories of recommendations to hide unneeded results.
+
+NOTE: The following procedure demonstrates how to set and remove *Category* filters; however, the procedure is applicable to any of the primary and secondary filters. 
+
+.Prerequisites
+You are logged into the Red Hat Hybrid Cloud Console.
+
+.Procedure
+. Navigate to the https://console.redhat.com/openshift/insights/advisor/recommendations[Red Hat Hybrid Cloud Console > OpenShift > Advisor recommendations] page.
+. Using the filter input fields above the list of recommendations, select the *Category* primary filter.
+. Expand the secondary filter dropdown and check the box next to each category of recommendation you want to view. Leave unnecessary categories unchecked.
+. Optionally, add additional filters to further refine the list.
+
+Only recommendations from the selected categories are shown in the list. 
+
+.Verification
+You can see the applied filters, including default filters, below the filter input dropdowns and above the list of recommendations.
+
+.Removing filters
+
+* You can remove applied filters individually, including the default filters, by clicking the *X* next to the filter, where applied filters are displayed.
+* You can remove only the filters that you applied, leaving the default filters intact, by clicking the *Remove filters* button.
```

Comment:
```suggestion
You can apply multiple filters to the list of recommendations and remove them individually by performing one of the following actions:
* Click the clear icon next to the filter.
* Click *Reset filters* to return to the default view without any manually applied filters.
```
I do not see the *Remove filters* button in the GUI. Please check the second bullet point for correctness.